spec/services/ssm_parameters_spec.rb

```diff
@@ -13,9 +13,17 @@
   describe "#put_parameter" do
     let(:parameter_value) { "test123"}
 
-    it "put ssm parameteRs" do
+    it "put ssm parameters" do
       response = described_class.new(district, name).put_parameter(parameter_value)
       expect(response.version).to eq 0
     end
   end
+
+  describe "#delete_parameter" do
+    it "delete ssm parameter" do
+      response = described_class.new(district, name).delete_parameter
+      expect(response.deleted_parameters).to eq []
+      expect(response.invalid_parameters).to eq []
```

Comment:
We should also check here that we are sending out a delete request to AWS. (just mocking AWS will do, we don't have to really send it)

Reply:
@davidsiaw Thank you for the reviews!
updated here https://github.com/degica/barcelona/pull/679/commits/a29189515cec43ee688f371c151b6fb13b1be68c
currently the request is already mocked so it should be ok.